src/blocks/gif/edit.js

```diff
@@ -156,7 +139,7 @@ class Edit extends Component {
 
 								// Disable reason: Image itself is not meant to be
 								// interactive, but should direct focus to block
-								// eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+								// eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
 								const img = <img src={ url } alt={ defaultedAlt } onClick={ this.onImageClick } />;
```

Comment:
We shouldn't ignore the eslint warning here. Can we rewrite this to make an anchor or button element so that it's properly accessible?